apps/verify/behaviours/send-verification-email.js

```diff
@@ -10,6 +10,7 @@ const templateId = config.govukNotify.userAuthTemplateId;
 const baseUrl = `${config.saveService.host}:${config.saveService.port}/saved_applications`;
 const tokenGenerator = require('../../../db/save-token');
 const _ = require('lodash');
+const { error } = require('jquery');
```

Comment:
Is `error` used here?

Reply:
I've pushed updates to this file by accident. Will remove.

Reply:
Have removed :)